SubBot/Services/Implementations/SimpleCommandsHandler.cs

```diff
@@ -37,7 +37,7 @@ private Task InitializeCommandService()
             try { (_commands as IDisposable)?.Dispose(); } catch { }
 
             CommandOptions options = this._commandOptions.CurrentValue;
-            CommandServiceConfig config = new CommandServiceConfig();
+            CommandServiceConfig config = new();
```

Comment:
Not sure if I like this, but it's ok - worst case will change this one back.

Reply:
Alright, it should be back in order.